engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/db/PeriodicProcessDeploymentsTable.scala

```diff
@@ -26,13 +26,15 @@ trait PeriodicProcessDeploymentsTableFactory extends PeriodicProcessesTableFacto
 
     def runAt: Rep[LocalDateTime] = column[LocalDateTime]("run_at", NotNull)
 
+    def scheduleName: Rep[String] = column[String]("schedule_name", NotNull)
+
     def deployedAt: Rep[Option[LocalDateTime]] = column[Option[LocalDateTime]]("deployed_at")
 
     def completedAt: Rep[Option[LocalDateTime]] = column[Option[LocalDateTime]]("completed_at")
 
     def status: Rep[PeriodicProcessDeploymentStatus] = column[PeriodicProcessDeploymentStatus]("status", NotNull)
 
-    override def * : ProvenShape[PeriodicProcessDeploymentEntity] = (id, periodicProcessId, createdAt, runAt, deployedAt, completedAt, status) <>
+    override def * : ProvenShape[PeriodicProcessDeploymentEntity] = (id, periodicProcessId, createdAt, runAt, scheduleName, deployedAt, completedAt, status) <>
```

Comment:
I wonder about deployments.. Right now only periodic process has deployments? What about normal process? - I know that is not part of your changes but.. :)